packages/driver/src/cy/commands/querying/querying.ts

```diff
@@ -142,13 +142,21 @@ function getAlias (selector, log, cy) {
 }
 
 export default (Commands, Cypress, cy, state) => {
+  function validateTimeoutFromOpts (options: GetOptions | ContainsOptions | ShadowOptions = {}) {
+    if (_.isPlainObject(options) && options.hasOwnProperty('timeout') && !_.isFinite(options.timeout)) {
+      $errUtils.throwErr(new Error(`contains invalid timeout - ${options.timeout}`))
```

Comment:
Typically we keep error messages within our [`error_messages.ts` file](https://github.com/cypress-io/cypress/blob/a11y-fix-3/packages/driver/src/cypress/error_messages.ts#L144) and call into that to throw the error message (for simplicity). You can see an example of this in line 153 of this same file. Could you refactor this to put the message there?

For the error message, to match other error messages to give the user all the info they need to fix it, I would suggest: 

```
{{cmd}} only accepts a number for its `timeout` option. You passed: \`{{timeout}}\`
```

So that for the user it will print:

```
`cy.get` only accepts a number for its `timeout` option. You passed: `abc`
```

Reply:
Thanks for the info, added new error msgs 👍

Reply:
This is addressed.